### PR TITLE
[#161915009] Handle back button on session expired

### PR DESCRIPTION
**Pull request by fpersico**
I have also added a button in the ProfileMainScreen to Simulate session expired.

sessionexpired

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#161915009](https://www.pivotaltracker.com/story/show/161915009): All'apertura trovo sempre questa schermata. Non funziona né il back né l'annulla

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: locales/it/index.yml [hunk 1]
```diff
@@ -69,6 +69,7 @@ profile:
     screenSubtitle: "Qui trovi informazioni e dati legati al tuo account: identità, sicurezza, tuoi
       dati personali, etc."
     logout: Logout/Esci
+    simulateSessionExpired: Simula sessione scaduta
```
Comment (cloudify) [inline locales/it/index.yml]:
```suggestion
    forgetCurrentSession: Dimentica sessione corrente
```

File: locales/en/index.yml [hunk 2]
```diff
@@ -67,6 +67,7 @@ profile:
     screenTitle: Profile
     screenSubtitle: Here you will find information and data related to security, identity, account and personal data.
     logout: Logout/Exit
+    simulateSessionExpired: Simulate session expired
```
Comment (cloudify) [inline locales/en/index.yml]:
```suggestion
    forgetCurrentSession: Forget current session
```

File: ts/store/actions/authentication.ts [hunk 3]
```diff
@@ -44,6 +44,10 @@ export const sessionExpired = createStandardAction("SESSION_EXPIRED")();
 
 export const sessionInvalid = createStandardAction("SESSION_INVALID")();
 
+export const resetAuthentication = createStandardAction(
```
Comment (cloudify) [inline ts/store/actions/authentication.ts]:
```suggestion
export const forgetCurrentSession = createStandardAction(
```

File: ts/store/actions/authentication.ts [hunk 4]
```diff
@@ -44,6 +44,10 @@ export const sessionExpired = createStandardAction("SESSION_EXPIRED")();
...
+  "AUTHENTICATION_RESET"
```
Comment (cloudify) [inline ts/store/actions/authentication.ts]:
```suggestion
  "SESSION_FORGET"
```

**Pull request merged**